### ISSUE TITLE
Modded Structures in Infinite Dimensions (question)

### ISSUE THREAD

**Issue by Cuplex17** [LeraRiemann/ProjectInfinity#42]
Is it possible to have modded structures generate in the new dimensions?
If so, what configs or datapacks do I need to add.
Thanks!

**Comment by LeraRiemann** [comment]
it is technically possible, however the (data format for it is undocumented and a bit complicated. you basically have to find the structure's own definition json and strip it of some data. reworking this is in my long term bin

for now, if you have particular structure(s) in mind, i can help you with making those config files myself, i think that'd be easier

**Comment by Cuplex17** [comment]
I see, thank you so much though.
I found a really bodgy way to do it currently. I made a datapack that replaces all vanilla structure-sets to include modded structures; this meant removing those structure-sets from all the other mods lol. It works but really tiresome. 
Eventually I think it's better if the config allows you to specify configured-features yourself instead of having hardcoded ones as it does now. 

Also found lots of other bugs that I will create issues for shortly.
